fetch-api/js/app.js

```diff
@@ -0,0 +1,70 @@
+const select = document.getElementById('breeds');
+const card = document.querySelector('.card');
+
+select.addEventListener('change', fetchBreedImage);
+card.addEventListener('click', fetchBreedImage);
+document.querySelector('form').addEventListener('submit', postInputFieldData);
+
+Promise.all([
+    fetchData('https://dog.ceo/api/breeds/image/random'),
+    fetchData('https://dog.ceo/api/breeds/list')
+])
+    .then(data => {
+        generateImage(data[0].message);
+        generateBreedOptions(data[1].message);
+    });
+
+//Functions:
+function checkStatus(response) {
+    if (response.ok) {
+        return Promise.resolve(response);
+    }
+    return Promise.reject(new Error(response.statusText));
+}
+
+function generateImage(data) {
+    card.innerHTML = `
+        <img src='${data}' alt>
+        <p>Click to view images of ${select.value}s</p>
+        `;
+}
+
+function generateBreedOptions(data) {
```

Comment:
What about renaming to `generateBreedOptionsElement` ?

I like that you used the same prefix though - 'generate'.

Reply:
Done.